### PR TITLE
l10n i18n OC update

### PR DESCRIPTION
**Pull request by Quenty31**
Normally it should be complete.

### CODE REVIEW

**Comment by Quenty31** [comment]
If you could tell me where I should change something, I would do it straight away :)

**Comment by ykzts** [comment]
It may be better to translate by [Weblate](https://weblate.joinmastodon.org).

**Comment by Quenty31** [comment]
Yes next time I'll do so!
But for the moment how to fix this PR? I can't read the log :(

File: config/locales/oc.yml [hunk 1]
```diff
@@ -503,12 +514,42 @@ oc:
...
+    generate: Generar
+    max_uses:
+      one: 1 persona
+      other: "%{count} personas"
+    max_uses_prompt: Cap limit
+    prompt: Generatz e partejatz los ligams per donar accès a aquesta instància
+    table:
+      expires_at: Expirats
+      uses: Usatges
+    title: Covidar de monde
```
Comment (ykzts) [inline config/locales/oc.yml]: remove trailing whitespaces.

**Comment by Quenty31** [comment]
It looks ok now @ykzts doesn't it?

**Review by Gargron — APPROVED** [review]

**Pull request merged**